src/completion-provider.js

```diff
@@ -110,6 +116,28 @@ class CompletionProvider {
       (suggestions) => this.filterSuggestions(prefix, suggestions)
     );
   }
+
+  lookupWebpack(prefix) {
+    const projectPath = atom.project.getPaths()[0];
+    if (!projectPath) {
+      return Promise.resolve([]);
+    }
+
+    const webpackConfigFilename = atom.config.get('autocomplete-modules.webpackConfigFilename');
+    const webpackConfigPath = path.join(projectPath, webpackConfigFilename);
+    const vendors = atom.config.get('autocomplete-modules.vendors');
+
+    const webpackConfig = require(webpackConfigPath);
+    let moduleSearchPaths = get(webpackConfig, 'resolve.modulesDirectories', []);
```

Comment:
You are right about plain object, but webpack config is still javascript file, so you can use es2015 classes or arrow-functions which could be unsupported in some atom releases.